src/instrument/configs/iconfig.yml

```
@@ -63,8 +67,8 @@ APS_DEVICES_FILE: devices_aps_only.yml
 OPHYD:
     ### Control layer for ophyd to communicate with EPICS.
     ### Default: PyEpics
-    ### Choices: "PyEpics" or "caproto"
-    # CONTROL_LAYER: caproto
+    ### Choices: "PyEpics" or "caproto" # caproto is not yet supported
```

Why?  We need to keep the ability to make a *field replacement*.  Two times in the past five years at APS.

---

Perhaps this might say
```suggestion
    ### Choices: "PyEpics" or "caproto" # caproto is not yet tested with this package
```